2023/10/19/main.go

```diff
@@ -0,0 +1,33 @@
+// Daily Joel 2023-10-19
+//
+// Variable declaration and initialization, variable scopes,
+// and variadic parameters.
+package main
+
+import (
+	"fmt"
+)
+
+func concatenate(s ...string) string {
+	out := ""
+	for _, ss := range s {
+		out += ss
+	}
+	return out
+}
```

Comment:
Concatanate is here a function that can take a varying amount of parameters, with one declared parameter, s. The return type is a string and we'll return the variable out

Out is an empty string

For every iteration of the for range loop we will add every item thrown at it to the out variable.